tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -33,7 +46,11 @@ Next, we need to add the NServiceBus NuGet package as a dependency. From the [Nu
 Install-Package NServiceBus -ProjectName ClientUI
 ```
 
-This adds a reference to the NServiceBus.Core assembly to the project. With the proper dependencies in place, we're ready to start writing code.
+This adds a reference to the NServiceBus.Core assembly to the project.
+
+NOTE: If you are using the SQL Server transport, you also need to install the `NServiceBus.SqlServer` package before continuing. See [Using the SQL Server transport - Adding the NuGet package](using-sql-transport.md#modifying-each-endpoint-adding-the-nuget-package) for more details.
```

Comment:
Only multi-paragraph NOTEs (or ones containing lists or other multi-line markup) need the {{ }} bracers.